app/assets/javascripts/initializers/initializeArticleReactions.js

```diff
@@ -27,6 +27,25 @@ function setReactionCount(reactionName, newCount) {
   }
 }
 
+function setSumReactionCount(counts) {
+  let totalCountObj = document.getElementById('reaction_total_count');
+  if (totalCountObj && counts.length > 2) {
+    let sum = 0;
+    for (let i in counts) {
+      if (counts[i]['category'] != 'readinglist') {
+        sum += counts[i]['count'];
+      }
+    }
+    totalCountObj.textContent = sum;
+  }
+}
+function showCommentCount() {
+  let commentCountObj = document.getElementById('reaction-number-comment');
```

Comment:
Suggestion: The id naming looks a bit inconsistent, in above function we are writing id as `reaction_total_count` and here its `reaction-number-comment`
I think both of these should be `-` spearated.

Reply:
Yeah, I agree with that. Since that was already in place, I think I'll consider that refactor out of scope, but agree going in and treating that as a chance to make things more consistent and standard is the right idea.